pages/api/v1/posts/on-chain-post.ts

```diff
@@ -79,6 +78,13 @@ export interface IReactions {
 	};
 }
 
+export interface IPIPsVoting {
+	balance: null | string;
```

Comment:
instead of storing null, keep balance and voter optional

Reply:
y value subsquid se hi null aari rhi h (balance: null)